HTML/Lyrical Nonsense.html

```diff
@@ -1,82 +1,118 @@
 <!DOCTYPE html>
 <html>
-<head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="initial-scale=1">
-    <title>Lyrical Nonsense — Keep Talking and Nobody Explodes Module</title>
-    <link rel="stylesheet" type="text/css" href="css/font.css">
-    <link rel="stylesheet" type="text/css" href="css/normalize.css">
-    <link rel="stylesheet" type="text/css" href="css/main.css">
-    <script src="js/ktane-utils.js"></script>
-    <style>
-        table {
-            margin: 1em auto;
-        }
-    </style>
-</head>
-<body>
-    <div class="section">
-        <div class="page page-bg-01">
-            <div class="page-header">
-                <span class="page-header-doc-title">Keep Talking and Nobody Explodes Mod</span>
-                <span class="page-header-section-title">Lyrical Nonsense</span>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="initial-scale=1">
+        <title>Lyrical Nonsense — Keep Talking and Nobody Explodes Module</title>
+        <link rel="stylesheet" type="text/css" href="css/font.css">
+        <link rel="stylesheet" type="text/css" href="css/normalize.css">
+        <link rel="stylesheet" type="text/css" href="css/main.css">
+        <script src="js/ktane-utils.js"></script>
+        <style>
+            table {
+                margin: 1em auto;
+            }
+        </style>
+    </head>
+    <body>
+        <div id="ManualContent">
```

Comment:
The spacing in this document and this line tells me that the repo copy was not edited. Edits need to be made to the repo copy of manuals. Please fix the file or ask the author to do this.

Reply:
Im gonna undo the change in lyrical nonsense dont wanna deal with it again.